pkg/oc/clusterup/run_self_hosted.go

```diff
@@ -506,9 +348,13 @@ func (c *ClusterUpConfig) startKubelet(out io.Writer, masterConfigDir, nodeConfi
 		}
 		actualKubeletFlags = append(actualKubeletFlags, curr)
 	}
+
+	glog.Info("kubelet flags = %s", strings.Join(actualKubeletFlags, " "))
+
 	container.Args = append(actualKubeletFlags, "--pod-manifest-path=/var/lib/origin/pod-manifests")
 	container.Args = append(container.Args, "--file-check-frequency=1s")
 	container.Args = append(container.Args, "--cluster-dns=172.30.0.2")
+	container.Args = append(container.Args, "--node-labels=node-role.kubernetes.io/master=")
```

Comment:
probably have to be replaced by oc label but that will require some polling on oc cluster up side for the api readiness? also this is an alpha feature